jukepoot/puppet_parser.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+"""A parser of /var/log/puppet.log.
+
+Imports argparse for parsing arguments and then the script
+accepts file path and year as optional argument. 
+"""
+
+
+import argparse 
+from datetime import datetime
+import dateutil.parser
+import re
+import collections
+import os
+
+
+def main():
+	parser = argparse.ArgumentParser()
+	parser.add_argument(help="parse the given filename", type=str, dest="filename", action="store")
```

Comment:
line too long (Applies to others as well), please fix those

parser.add_argument(
    help="parse....", ..., action="store")